### DISCUSSION TITLE
How to solve the annoying delay?

### DISCUSSION THREAD

**Discussion by XilyFeAAAA** [JanDeDobbeleer/oh-my-posh#2841] [Troubleshoot]
Every time I had to wait about 10 seconds before next prompt.
After viewing the official FAQ, i used "oh-my-posh debug" to check out what was going wrong.And the problem seemed to be that the HTTPRequest cost too much time as follow.
![93ZK)NBGIC(BD$YD5H$1 D8](https://user-images.githubusercontent.com/114482179/192498894-af045292-349a-4332-9ddb-1a34a1f3d8d2.png)
Is it because I'm in China now or  there are some problems with config files. And how to solve it

**Comment by XilyFeAAAA** [comment]
I think this could be because oh-my-posh didn't recognize my theme data so it turned to download from the website
![}RXC(`0VONFRM07IO1FC Z3](https://user-images.githubusercontent.com/114482179/192503682-f3594305-356c-4b47-9bea-92331d6ae6a6.png)
![UR)CD0BQ2@N8G644SZB~%DX](https://user-images.githubusercontent.com/114482179/192503730-6ae0450c-a4a7-407c-aed4-26101197cf8c.png)

**Reply by JanDeDobbeleer** [reply]
@XilyFeAAAA correct. You can validate the path (also looking at the environment variable). That said, you're still using the module so you're probably also out of date quite a lot. Follow [this guide](https://ohmyposh.dev/docs/migrating) to move to the executable.

**Reply by XilyFeAAAA** [reply]
tks,i have solved this problem!
